bentoml/_internal/runner/runner.py

```diff
@@ -209,6 +211,13 @@ def init_local(self, quiet: bool = False) -> None:
         if not quiet:
             logger.warning("'Runner.init_local' is for debugging and testing only")
 
+        if self.service_ref:
+            service = self.service_ref()
+            if service and service.bento:
+                with BentoMLContainer.model_store.patch(service.bento._model_store):  # type: ignore
+                    self._init_local()
+                return
```

Comment:
We dot not expect an `else` case here, right? Should we raise?

Reply:
if user run this in a separate Runner or distributedly, the behaviour is the same as before.

we only want to change the model store if it runs inside the bento.

Reply:
hense, raise is not desirable here.

Reply:
Understood the distributed case, what do you mean by running this in a separate Runner? Do you mean a separate Service?

Reply:
separate runner.